### ISSUE TITLE
Fix pytest the remaining 17 warnings

### ISSUE THREAD

**Issue by tinatn29** [regro/regolith#1208]
### Problem

17 warnings remain when running `pytest`

src/regolith/tests/test_builders.py::test_builder[fs-recent-collabs]
src/regolith/tests/test_builders.py::test_builder_python[recent-collabs-fs]
  /Users/Baitong/opt/anaconda3/envs/rego_env_311_new/lib/python3.11/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Data Validation extension is not supported and will be removed
    warn(msg)

src/regolith/tests/test_builders.py::test_builder[fs-recent-collabs]
src/regolith/tests/test_builders.py::test_builder_python[recent-collabs-fs]
  /Users/Baitong/opt/anaconda3/envs/rego_env_311_new/lib/python3.11/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed
    warn(msg)

src/regolith/tests/test_validate.py::test_validate_python
src/regolith/tests/test_validate.py::test_validate_bad_python
src/regolith/tests/test_validators.py::test_validation[grants]
src/regolith/tests/test_validators.py::test_exemplars[grants]
  /Users/Baitong/My Drive/BGPostdoc/regolith/regolith/src/regolith/schemas.py:187: UserWarning: "researcher" is not in the preferred entries for "position", please consider changing this entry to conform or add this to the ``eallowed`` field in the schema.
    warn(

src/regolith/tests/test_validate.py::test_validate_python
  /Users/Baitong/My Drive/BGPostdoc/regolith/regolith/src/regolith/schemas.py:187: UserWarning: "" is not in the preferred entries for "recommendation", please consider changing this entry to conform or add this to the ``eallowed`` field in the schema.
    warn(

src/regolith/tests/test_validate.py::test_validate_python
src/regolith/tests/test_validate.py::test_validate_bad_python
src/regolith/tests/test_validators.py::test_validation[expenses]
src/regolith/tests/test_validators.py::test_exemplars[expenses]
  /Users/Baitong/My Drive/BGPostdoc/regolith/regolith/src/regolith/schemas.py:187: UserWarning: "bad_status" is not in the preferred entries for "status", please consider changing this entry to conform or add this to the ``eallowed`` field in the schema.
    warn(

src/regolith/tests/test_validate.py::test_validate_python
src/regolith/tests/test_validate.py::test_validate_bad_python
src/regolith/tests/test_validators.py::test_validation[people]
src/regolith/tests/test_validators.py::test_exemplars[people]
  /Users/Baitong/My Drive/BGPostdoc/regolith/regolith/src/regolith/schemas.py:187: UserWarning: "janitor" is not in the preferred entries for "position", please consider changing this entry to conform or add this to the ``eallowed`` field in the schema.
    warn(

### Proposed solution
Edit the schema to match or edit the entry in `exemplars.json`? We wonder if some of these might have been "wrong" entries by design to test the warnings... (e.g. our schemas may not need to include "bad_status" or "janitor")

**Comment by tinatn29** [comment]
@sbillinge these warnings show entries that do not match the schemas^, do we edit the schemas to match? A number of these look like test cases that were designed to trigger the warnings?

**Comment by bobleesj** [comment]
@tinatn29 a quick question, do all warnings originate from this unseen data in the schema ?

**Comment by tinatn29** [comment]
@bobleesj the first (top) two warnings are different: Data Validation extension not supported and Unknown extension not supported, but I don't know what they mean...

**Comment by bobleesj** [comment]
Yes, if you could make separate issues for those, that would be good. So that our PRs become modular.

**Comment by bobleesj** [comment]
A title sth along the line of `Fix 17 out of xx pytest warnings - data schema key ... blah blah blah`

**Comment by tinatn29** [comment]
ok! edited this one and will create another one for the other warnings. It will fix < 17 warnings because some of them are the different warnings 😅 so I left the number out for now

**Comment by sbillinge** [comment]
I fixed most of the schema issues.  Some that were left may have been coming from the tests.  I will have to look.  Don't spend too much time on these schema warnings until I look at them.
